tests/baselines/reference/exportEqualsAmd.js

```diff
@@ -4,6 +4,6 @@ export = { ["hi"]: "there" };
 //// [exportEqualsAmd.js]
 define(["require", "exports"], function (require, exports) {
     "use strict";
-    return (_a = {}, _a["hi"] = "there", _a);
+    return _a = {}, _a["hi"] = "there", _a;

```

Comment:
Yes. The node factories will automatically add parenthesis if they are needed. As a result, many places where we explicitly added parentheses now instead leverage this behavior. In this instance, the `createReturn` factory does not need to parenthesize, so they are not added.